pup.py

```diff
@@ -251,7 +280,15 @@ def main():
         mnm.start_http_server(port=9126)
         loop.set_default_executor(thread_pool_executor)
         loop.create_task(CONSUMER.get_callback(consume)())
-        loop.create_task(PRODUCER.get_callback(make_producer(produce_queue))())
+        loop.create_task(PRODUCER.get_callback(make_responder(produce_queue))())
+        loop.create_task(
```

Comment:
this is a nitpick, and I know it's more than 80 characters in a line, but it looks super funky to have one of these broken down and the others on one line.